### ISSUE TITLE
Visual indication of scrobbler errors

### ISSUE THREAD

**Issue by inverse** [web-scrobbler/web-scrobbler#3187]
When sending data to last.fm sometimes it responses with `ignoredMessage` values meaning the data was ignored on last.fm side. Currently we don't error giving users confusion under such cases.

Example response

```
Last.fm: track.updatenowplaying response:
{
  "nowplaying": {
    "artist": {
      "corrected": "0",
      "#text": "Current 93"
    },
    "track": {
      "corrected": "0",
      "#text": "Killy Kill Killy *A Fire Sermon* (Previously Unreleased)"
    },
    "ignoredMessage": {
      "code": "1",
      "#text": ""
    },
    "albumArtist": {
      "corrected": "0",
      "#text": "Various Artists (Fundraiser for Ukraine)"
    },
    "album": {
      "corrected": "0",
      "#text": "Slava Ukraini!"
    }
  }

```

API: https://www.last.fm/api/scrobbling#filtered-requests

We should map these and bubble up to the user some meaning behind these.

Identified in https://github.com/web-scrobbler/web-scrobbler/issues/3165#issuecomment-1065961360

**Comment by yayuyokitano** [comment]
Fix completed by #4145 
Once both are merged, this can be clsoed.

**Comment by yayuyokitano** [comment]
I forgor💀
